src/Event.php

```diff
@@ -101,6 +101,11 @@ final class Event implements \JsonSerializable
      */
     private $userContext;
 
+    /**
+     * @var Context An arbitrary mapping of additional contexts associated to this event
+     */
+    private $contexts;
```

Comment:
Please switch to `array<string, array<string, mixed>>`, see similar comment below for the explanation of why I'm requesting this change